plugin/shared/config.json

```diff
@@ -0,0 +1,65 @@
+{
+	"Description": "VMWare vSphere Shared Docker Volume plugin",
```

Comment:
I think it's better to use 1 json and just sed the difference before building plugin image.
Having 2 copies is error prone